src/translations/en-US.json

```diff
@@ -303,5 +303,15 @@
       "value": "Aggregations",
       "comment": "",
       "limit": 0
+   },
+   "visualizations.menu.aggregations.rows": {
+      "value": "Rows",
+      "comment": "Heading of row attributes list aggregation submenu of Table component. Selecting an attribute enables aggregation on selected attribute (adds 'total' or 'subtotal' row in attribute column of table).",
```

Comment:
I think we need to come up with something simpler for the translators. They are not so familiar with the domain. I think a comment like `"Row" here means table row` would do the job.